### PR TITLE
Doubleclick

### PR DESCRIPTION
**Pull request by olafur-palsson**
Here it is, the doubleClick feature implemented in the way that you requested. :)

### CODE REVIEW

File: app/components/TrackRow/index.js [hunk 1]
```diff
@@ -43,6 +55,15 @@ class TrackRow extends React.Component {
     );
   }
 
+  playTrack () {
+    console.log(this.getThumbnail(this.props.track));
```
Comment (nukeop) [inline app/components/TrackRow/index.js]: could you remove this debug statement?

**Comment by nukeop** [comment]
Thanks, looking good. Just one small change and it'll be ready to merge.

File: app/components/TrackRow/index.js (hunk 1 above)
Comment (olafur-palsson) [reply app/components/TrackRow/index.js]: It is

File: app/components/TrackRow/index.js [hunk 2]
```diff
@@ -1,18 +1,30 @@
...
 import TrackPopupContainer from '../../containers/TrackPopupContainer';
 import { formatDuration } from '../../utils';
 
 import styles from './styles.scss';
 
 class TrackRow extends React.Component {
-  getThumbnail(track) {
+  constructor () {
+    super();
+    this.state = { doubleClick: false };
```
Comment (nukeop) [inline app/components/TrackRow/index.js]: I think we won't need this in this component anymore, and it can be converted to a stateless one.

**Comment by nukeop** [comment]
One more thing I just noticed. I also tested the code and it works well.

**Comment by olafur-palsson** [comment]
Should be good now.

**Comment by nukeop** [comment]
Great, thank you.

**Comment by nukeop** [comment]
Your change broke the functionality, since `this` is unbound in `playTrack`. You'll need to bind it.

**Comment by olafur-palsson** [comment]
Sorry for taking long, I was on a vacation in Spain with no laptop.

**Comment by nukeop** [comment]
No problem, we don't have deadlines here. Thanks for the contribution.

**Pull request merged**